Synthesis failed for Google.Cloud.Monitoring.V3

Hello! Autosynth couldn't regenerate Google.Cloud.Monitoring.V3. :broken_heart:

Please investigate and fix this issue within 5 business days.  While it remains broken,
this library cannot be updated with changes to the Google.Cloud.Monitoring.V3 API, and the library grows
stale.

See https://github.com/googleapis/synthtool/blob/master/autosynth/TroubleShooting.md
for trouble shooting tips.

Here's the output from running `synth.py`:

```
95c4b4a3ffe4303f1f2984ccc304b2e0
2021-08-26 22:53:37,907 autosynth [DEBUG] > Running: git log -1 --pretty=%at 68e5bf3045d7b3ef55f473494a0cd4566491f899
2021-08-26 22:53:37,910 autosynth [DEBUG] > Running: git log -1 --pretty=%at a5b180f2f9fd9a48ad7ed799cbf438fa354bba32
2021-08-26 22:53:37,913 autosynth [DEBUG] > Running: git log -1 --pretty=%at d06d94e541c6405a2f12cc35bb60ecc1a7dfdb52
2021-08-26 22:53:37,917 autosynth [DEBUG] > Running: git log -1 --pretty=%at a0721b50dbc8692efefb75ffaab5d6ec10f62d0e
2021-08-26 22:53:37,920 autosynth [DEBUG] > Running: git log -1 --pretty=%at 061db160ed427e53884074d85061b59ce1702821
2021-08-26 22:53:37,923 autosynth [DEBUG] > Running: git log -1 --pretty=%at 7ae76714a266453c2e64a1faa75f156cbe7b6610
2021-08-26 22:53:37,926 autosynth [DEBUG] > Running: git log -1 --pretty=%at 9e617db5033d24bb8a47d6ac0d48de33b6f9ab16
2021-08-26 22:53:37,929 autosynth [DEBUG] > Running: git log -1 --pretty=%at ba30d8097582039ac4cc4e21b4e4baa426423075
2021-08-26 22:53:37,932 autosynth [DEBUG] > Running: git log -1 --pretty=%at 8c0e4a368056920eaa1039cd740a637c9d12cccb
2021-08-26 22:53:37,935 autosynth [DEBUG] > Running: git log -1 --pretty=%at 27600833c3d6e80cd676d5b54c54f89cb3f55665
2021-08-26 22:53:37,938 autosynth [DEBUG] > Running: git log -1 --pretty=%at cb637a6db2a0e406d656055a286ad921134c7d57
2021-08-26 22:53:37,942 autosynth [DEBUG] > Running: git log -1 --pretty=%at 8bf048db6962fffc98b3930de2b4044aa1d54957
2021-08-26 22:53:37,945 autosynth [DEBUG] > Running: git log -1 --pretty=%at 8a770c332426e30d7ce89de33d92eabd8835b8f0
2021-08-26 22:53:37,948 autosynth [DEBUG] > Running: git log -1 --pretty=%at 3cd708edbc97d8b0ceee6e667229e19f19249806
2021-08-26 22:53:37,951 autosynth [DEBUG] > Running: git log -1 --pretty=%at dd5050968965c901275e2d34d7e928c57028dfeb
2021-08-26 22:53:37,954 autosynth [DEBUG] > Running: git log -1 --pretty=%at cdc9747a73f6af2393ab51490fc3a3c11ff3620f
2021-08-26 22:53:37,957 autosynth [DEBUG] > Running: git log -1 --pretty=%at be31a88c169ec48f0862eb7632d0d514327d9948
2021-08-26 22:53:37,960 autosynth [DEBUG] > Running: git log -1 --pretty=%at 41e44ac10040bf2801c9a5c7e846e55029a02243
2021-08-26 22:53:37,964 autosynth [DEBUG] > Running: git log -1 --pretty=%at 5572865403528e706cd895b8817f0e54fae74c02
2021-08-26 22:53:37,967 autosynth [DEBUG] > Running: git log -1 --pretty=%at 410f323cdbbe047bbf4527ab453b89930917f4a8
2021-08-26 22:53:37,970 autosynth [DEBUG] > Running: git log -1 --pretty=%at 4006aa5cbd12a86550d45fb215511a98a697f48f
2021-08-26 22:53:37,973 autosynth [DEBUG] > Running: git log -1 --pretty=%at 4b6bb3dede843b93353dbebfee4d2758aeede96f
2021-08-26 22:53:37,976 autosynth [DEBUG] > Running: git log -1 --pretty=%at 9e36f4ec3accbe70c7d11e8242cf5f4e2a1e0957
2021-08-26 22:53:37,979 autosynth [DEBUG] > Running: git log -1 --pretty=%at 354d0763e7e4ce7ab370790ffba0c3c25b1420c3
2021-08-26 22:53:37,983 autosynth [DEBUG] > Running: git log -1 --pretty=%at ccb73479aebb76f13698907884269bb7a38b4207
2021-08-26 22:53:37,986 autosynth [DEBUG] > Running: git checkout e84109d6aa5b79595eb16fb9a14cc3cc4b76783a
Note: switching to 'e84109d6aa5b79595eb16fb9a14cc3cc4b76783a'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at e84109d6a Re-generated apis/Google.Cloud.CloudBuild.V1 to pick up changes in the API or client library generator. (#7040)
2021-08-26 22:53:38,009 autosynth [DEBUG] > Running: git checkout ccb73479aebb76f13698907884269bb7a38b4207
Note: switching to 'ccb73479aebb76f13698907884269bb7a38b4207'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at ccb73479 chore: regenerate API index
2021-08-26 22:53:38,024 autosynth [DEBUG] > Running: git branch -f autosynth-Google.Cloud.Monitoring.V3-42
2021-08-26 22:53:38,027 autosynth [DEBUG] > Running: git checkout autosynth-Google.Cloud.Monitoring.V3-42
Switched to branch 'autosynth-Google.Cloud.Monitoring.V3-42'
2021-08-26 22:53:38,050 autosynth [INFO] > Running synthtool
2021-08-26 22:53:38,050 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2021-08-26 22:53:38,050 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/Google.Cloud.Monitoring.V3/42/sponge_log.log
2021-08-26 22:53:38,052 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2021-08-26 22:53:38,274 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/google-cloud-dotnet/apis/Google.Cloud.Monitoring.V3/synth.py.
On branch autosynth-Google.Cloud.Monitoring.V3-42
nothing to commit, working tree clean
Skipping microgenerator fetch/build: already built, and running on Kokoro
Building existing version of Google.Cloud.Monitoring.V3 for compatibility checking
Generating Google.Cloud.Monitoring.V3
Building new version of Google.Cloud.Monitoring.V3 for compatibility checking
UptimeService.g.cs(93,245): error CS0234: The type or namespace name 'LongRunning' does not exist in the namespace 'Google' (are you missing an assembly reference?) [/home/kbuilder/.cache/synthtool/google-cloud-dotnet/apis/Google.Cloud.Monitoring.V3/Google.Cloud.Monitoring.V3/Google.Cloud.Monitoring.V3.csproj]
2021-08-26 22:53:56,360 synthtool [ERROR] > Failed executing /bin/bash generateapis.sh --check_compatibility Google.Cloud.Monitoring.V3:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1137, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1062, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 763, in invoke
    return __callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/google-cloud-dotnet/apis/Google.Cloud.Monitoring.V3/synth.py", line 26, in <module>
    hide_output = False)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '('/bin/bash', 'generateapis.sh', '--check_compatibility', 'Google.Cloud.Monitoring.V3')' returned non-zero exit status 1.
2021-08-26 22:53:56,388 autosynth [ERROR] > Synthesis failed
2021-08-26 22:53:56,388 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at e84109d6a Re-generated apis/Google.Cloud.CloudBuild.V1 to pick up changes in the API or client library generator. (#7040)
2021-08-26 22:53:56,463 autosynth [DEBUG] > Running: git checkout autosynth-Google.Cloud.Monitoring.V3
Switched to branch 'autosynth-Google.Cloud.Monitoring.V3'
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 359, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 191, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 339, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 68, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/34bf6240-40fd-4d04-aa5d-cd4c565f0711/targets/github%2Fsynthtool;config=default/tests;query=google-cloud-dotnet;failed=false).

Autosynth is still having trouble generating Google.Cloud.Monitoring.V3. :sob:

Please investigate and fix this issue within 5 business days.  While it remains broken,
this library cannot be updated with changes to the Google.Cloud.Monitoring.V3 API, and the library grows
stale.

See https://github.com/googleapis/synthtool/blob/master/autosynth/TroubleShooting.md
for trouble shooting tips.

Here's the output from running `synth.py`:

```
75ffaab5d6ec10f62d0e
2021-08-27 22:59:25,840 autosynth [DEBUG] > Running: git log -1 --pretty=%at 061db160ed427e53884074d85061b59ce1702821
2021-08-27 22:59:25,844 autosynth [DEBUG] > Running: git log -1 --pretty=%at 7ae76714a266453c2e64a1faa75f156cbe7b6610
2021-08-27 22:59:25,847 autosynth [DEBUG] > Running: git log -1 --pretty=%at 9e617db5033d24bb8a47d6ac0d48de33b6f9ab16
2021-08-27 22:59:25,851 autosynth [DEBUG] > Running: git log -1 --pretty=%at ba30d8097582039ac4cc4e21b4e4baa426423075
2021-08-27 22:59:25,855 autosynth [DEBUG] > Running: git log -1 --pretty=%at 8c0e4a368056920eaa1039cd740a637c9d12cccb
2021-08-27 22:59:25,858 autosynth [DEBUG] > Running: git log -1 --pretty=%at 27600833c3d6e80cd676d5b54c54f89cb3f55665
2021-08-27 22:59:25,862 autosynth [DEBUG] > Running: git log -1 --pretty=%at cb637a6db2a0e406d656055a286ad921134c7d57
2021-08-27 22:59:25,865 autosynth [DEBUG] > Running: git log -1 --pretty=%at 8bf048db6962fffc98b3930de2b4044aa1d54957
2021-08-27 22:59:25,869 autosynth [DEBUG] > Running: git log -1 --pretty=%at 8a770c332426e30d7ce89de33d92eabd8835b8f0
2021-08-27 22:59:25,872 autosynth [DEBUG] > Running: git log -1 --pretty=%at 3cd708edbc97d8b0ceee6e667229e19f19249806
2021-08-27 22:59:25,876 autosynth [DEBUG] > Running: git log -1 --pretty=%at dd5050968965c901275e2d34d7e928c57028dfeb
2021-08-27 22:59:25,879 autosynth [DEBUG] > Running: git log -1 --pretty=%at cdc9747a73f6af2393ab51490fc3a3c11ff3620f
2021-08-27 22:59:25,883 autosynth [DEBUG] > Running: git log -1 --pretty=%at be31a88c169ec48f0862eb7632d0d514327d9948
2021-08-27 22:59:25,886 autosynth [DEBUG] > Running: git log -1 --pretty=%at 41e44ac10040bf2801c9a5c7e846e55029a02243
2021-08-27 22:59:25,890 autosynth [DEBUG] > Running: git log -1 --pretty=%at 5572865403528e706cd895b8817f0e54fae74c02
2021-08-27 22:59:25,893 autosynth [DEBUG] > Running: git log -1 --pretty=%at 410f323cdbbe047bbf4527ab453b89930917f4a8
2021-08-27 22:59:25,896 autosynth [DEBUG] > Running: git log -1 --pretty=%at 4006aa5cbd12a86550d45fb215511a98a697f48f
2021-08-27 22:59:25,900 autosynth [DEBUG] > Running: git log -1 --pretty=%at 4b6bb3dede843b93353dbebfee4d2758aeede96f
2021-08-27 22:59:25,903 autosynth [DEBUG] > Running: git log -1 --pretty=%at 9e36f4ec3accbe70c7d11e8242cf5f4e2a1e0957
2021-08-27 22:59:25,906 autosynth [DEBUG] > Running: git log -1 --pretty=%at 354d0763e7e4ce7ab370790ffba0c3c25b1420c3
2021-08-27 22:59:25,910 autosynth [DEBUG] > Running: git log -1 --pretty=%at ccb73479aebb76f13698907884269bb7a38b4207
2021-08-27 22:59:25,913 autosynth [DEBUG] > Running: git log -1 --pretty=%at 848854523c886fbfb421096a550533be96e0a49d
2021-08-27 22:59:25,916 autosynth [DEBUG] > Running: git log -1 --pretty=%at 43095748d60f2c54d908a27c420fef8ba3c9dbeb
2021-08-27 22:59:25,919 autosynth [DEBUG] > Running: git log -1 --pretty=%at 677caf0b524ea611211f39269bd1ee124438fba7
2021-08-27 22:59:25,923 autosynth [DEBUG] > Running: git log -1 --pretty=%at 41635ad2f614a85ac6ef92605b3e90d051b11ac0
2021-08-27 22:59:25,926 autosynth [DEBUG] > Running: git log -1 --pretty=%at e9da6f8b469c52b83f900e820be30762e9e05c57
2021-08-27 22:59:25,930 autosynth [DEBUG] > Running: git checkout aff9821e78546f6e6f8235d0ae9c348be826e46f
Note: switching to 'aff9821e78546f6e6f8235d0ae9c348be826e46f'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at aff9821e7 feat: Add Canada Regions And Support compliance regime
2021-08-27 22:59:25,954 autosynth [DEBUG] > Running: git checkout e9da6f8b469c52b83f900e820be30762e9e05c57
Note: switching to 'e9da6f8b469c52b83f900e820be30762e9e05c57'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at e9da6f8b feat: support version reporting API
2021-08-27 22:59:25,970 autosynth [DEBUG] > Running: git branch -f autosynth-Google.Cloud.Monitoring.V3-47
2021-08-27 22:59:25,974 autosynth [DEBUG] > Running: git checkout autosynth-Google.Cloud.Monitoring.V3-47
Switched to branch 'autosynth-Google.Cloud.Monitoring.V3-47'
2021-08-27 22:59:25,998 autosynth [INFO] > Running synthtool
2021-08-27 22:59:25,998 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2021-08-27 22:59:25,998 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/Google.Cloud.Monitoring.V3/47/sponge_log.log
2021-08-27 22:59:26,000 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2021-08-27 22:59:26,240 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/google-cloud-dotnet/apis/Google.Cloud.Monitoring.V3/synth.py.
On branch autosynth-Google.Cloud.Monitoring.V3-47
nothing to commit, working tree clean
Skipping microgenerator fetch/build: already built, and running on Kokoro
Building existing version of Google.Cloud.Monitoring.V3 for compatibility checking
Generating Google.Cloud.Monitoring.V3
Building new version of Google.Cloud.Monitoring.V3 for compatibility checking
UptimeService.g.cs(93,245): error CS0234: The type or namespace name 'LongRunning' does not exist in the namespace 'Google' (are you missing an assembly reference?) [/home/kbuilder/.cache/synthtool/google-cloud-dotnet/apis/Google.Cloud.Monitoring.V3/Google.Cloud.Monitoring.V3/Google.Cloud.Monitoring.V3.csproj]
2021-08-27 22:59:45,464 synthtool [ERROR] > Failed executing /bin/bash generateapis.sh --check_compatibility Google.Cloud.Monitoring.V3:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1137, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1062, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 763, in invoke
    return __callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/google-cloud-dotnet/apis/Google.Cloud.Monitoring.V3/synth.py", line 26, in <module>
    hide_output = False)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '('/bin/bash', 'generateapis.sh', '--check_compatibility', 'Google.Cloud.Monitoring.V3')' returned non-zero exit status 1.
2021-08-27 22:59:45,494 autosynth [ERROR] > Synthesis failed
2021-08-27 22:59:45,495 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at aff9821e7 feat: Add Canada Regions And Support compliance regime
2021-08-27 22:59:45,559 autosynth [DEBUG] > Running: git checkout autosynth-Google.Cloud.Monitoring.V3
Switched to branch 'autosynth-Google.Cloud.Monitoring.V3'
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 359, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 191, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 339, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 68, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/54751963-61c4-4a7f-88da-d756cb349895/targets/github%2Fsynthtool;config=default/tests;query=google-cloud-dotnet;failed=false).

Autosynth is still having trouble generating Google.Cloud.Monitoring.V3. :sob:

Please investigate and fix this issue within 5 business days.  While it remains broken,
this library cannot be updated with changes to the Google.Cloud.Monitoring.V3 API, and the library grows
stale.

See https://github.com/googleapis/synthtool/blob/master/autosynth/TroubleShooting.md
for trouble shooting tips.

Here's the output from running `synth.py`:

```
osynth [DEBUG] > Running: git log -1 --pretty=%at 061db160ed427e53884074d85061b59ce1702821
2021-08-28 08:52:01,990 autosynth [DEBUG] > Running: git log -1 --pretty=%at 7ae76714a266453c2e64a1faa75f156cbe7b6610
2021-08-28 08:52:01,993 autosynth [DEBUG] > Running: git log -1 --pretty=%at 9e617db5033d24bb8a47d6ac0d48de33b6f9ab16
2021-08-28 08:52:01,996 autosynth [DEBUG] > Running: git log -1 --pretty=%at ba30d8097582039ac4cc4e21b4e4baa426423075
2021-08-28 08:52:01,999 autosynth [DEBUG] > Running: git log -1 --pretty=%at 8c0e4a368056920eaa1039cd740a637c9d12cccb
2021-08-28 08:52:02,003 autosynth [DEBUG] > Running: git log -1 --pretty=%at 27600833c3d6e80cd676d5b54c54f89cb3f55665
2021-08-28 08:52:02,006 autosynth [DEBUG] > Running: git log -1 --pretty=%at cb637a6db2a0e406d656055a286ad921134c7d57
2021-08-28 08:52:02,009 autosynth [DEBUG] > Running: git log -1 --pretty=%at 8bf048db6962fffc98b3930de2b4044aa1d54957
2021-08-28 08:52:02,012 autosynth [DEBUG] > Running: git log -1 --pretty=%at 8a770c332426e30d7ce89de33d92eabd8835b8f0
2021-08-28 08:52:02,015 autosynth [DEBUG] > Running: git log -1 --pretty=%at 3cd708edbc97d8b0ceee6e667229e19f19249806
2021-08-28 08:52:02,018 autosynth [DEBUG] > Running: git log -1 --pretty=%at dd5050968965c901275e2d34d7e928c57028dfeb
2021-08-28 08:52:02,021 autosynth [DEBUG] > Running: git log -1 --pretty=%at cdc9747a73f6af2393ab51490fc3a3c11ff3620f
2021-08-28 08:52:02,024 autosynth [DEBUG] > Running: git log -1 --pretty=%at be31a88c169ec48f0862eb7632d0d514327d9948
2021-08-28 08:52:02,027 autosynth [DEBUG] > Running: git log -1 --pretty=%at 41e44ac10040bf2801c9a5c7e846e55029a02243
2021-08-28 08:52:02,031 autosynth [DEBUG] > Running: git log -1 --pretty=%at 5572865403528e706cd895b8817f0e54fae74c02
2021-08-28 08:52:02,034 autosynth [DEBUG] > Running: git log -1 --pretty=%at 410f323cdbbe047bbf4527ab453b89930917f4a8
2021-08-28 08:52:02,037 autosynth [DEBUG] > Running: git log -1 --pretty=%at 4006aa5cbd12a86550d45fb215511a98a697f48f
2021-08-28 08:52:02,040 autosynth [DEBUG] > Running: git log -1 --pretty=%at 4b6bb3dede843b93353dbebfee4d2758aeede96f
2021-08-28 08:52:02,043 autosynth [DEBUG] > Running: git log -1 --pretty=%at 9e36f4ec3accbe70c7d11e8242cf5f4e2a1e0957
2021-08-28 08:52:02,046 autosynth [DEBUG] > Running: git log -1 --pretty=%at 354d0763e7e4ce7ab370790ffba0c3c25b1420c3
2021-08-28 08:52:02,050 autosynth [DEBUG] > Running: git log -1 --pretty=%at ccb73479aebb76f13698907884269bb7a38b4207
2021-08-28 08:52:02,053 autosynth [DEBUG] > Running: git log -1 --pretty=%at 848854523c886fbfb421096a550533be96e0a49d
2021-08-28 08:52:02,056 autosynth [DEBUG] > Running: git log -1 --pretty=%at 43095748d60f2c54d908a27c420fef8ba3c9dbeb
2021-08-28 08:52:02,059 autosynth [DEBUG] > Running: git log -1 --pretty=%at 677caf0b524ea611211f39269bd1ee124438fba7
2021-08-28 08:52:02,062 autosynth [DEBUG] > Running: git log -1 --pretty=%at 41635ad2f614a85ac6ef92605b3e90d051b11ac0
2021-08-28 08:52:02,065 autosynth [DEBUG] > Running: git log -1 --pretty=%at e9da6f8b469c52b83f900e820be30762e9e05c57
2021-08-28 08:52:02,069 autosynth [DEBUG] > Running: git checkout 6b21a68d70a8bf416d2da830ca475a09d3e193b2
Note: switching to '6b21a68d70a8bf416d2da830ca475a09d3e193b2'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at 6b21a68d7 docs: clarified some LRO types docs: fixed some wrong update mask descriptions
2021-08-28 08:52:02,093 autosynth [DEBUG] > Running: git checkout e9da6f8b469c52b83f900e820be30762e9e05c57
Note: switching to 'e9da6f8b469c52b83f900e820be30762e9e05c57'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at e9da6f8b feat: support version reporting API
2021-08-28 08:52:02,109 autosynth [DEBUG] > Running: git branch -f autosynth-Google.Cloud.Monitoring.V3-47
2021-08-28 08:52:02,113 autosynth [DEBUG] > Running: git checkout autosynth-Google.Cloud.Monitoring.V3-47
Switched to branch 'autosynth-Google.Cloud.Monitoring.V3-47'
2021-08-28 08:52:02,136 autosynth [INFO] > Running synthtool
2021-08-28 08:52:02,136 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2021-08-28 08:52:02,136 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/Google.Cloud.Monitoring.V3/47/sponge_log.log
2021-08-28 08:52:02,139 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2021-08-28 08:52:02,393 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/google-cloud-dotnet/apis/Google.Cloud.Monitoring.V3/synth.py.
On branch autosynth-Google.Cloud.Monitoring.V3-47
nothing to commit, working tree clean
Skipping microgenerator fetch/build: already built, and running on Kokoro
Building existing version of Google.Cloud.Monitoring.V3 for compatibility checking
Generating Google.Cloud.Monitoring.V3
Building new version of Google.Cloud.Monitoring.V3 for compatibility checking
UptimeService.g.cs(93,245): error CS0234: The type or namespace name 'LongRunning' does not exist in the namespace 'Google' (are you missing an assembly reference?) [/home/kbuilder/.cache/synthtool/google-cloud-dotnet/apis/Google.Cloud.Monitoring.V3/Google.Cloud.Monitoring.V3/Google.Cloud.Monitoring.V3.csproj]
2021-08-28 08:52:20,417 synthtool [ERROR] > Failed executing /bin/bash generateapis.sh --check_compatibility Google.Cloud.Monitoring.V3:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1137, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1062, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 763, in invoke
    return __callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/google-cloud-dotnet/apis/Google.Cloud.Monitoring.V3/synth.py", line 26, in <module>
    hide_output = False)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '('/bin/bash', 'generateapis.sh', '--check_compatibility', 'Google.Cloud.Monitoring.V3')' returned non-zero exit status 1.
2021-08-28 08:52:20,445 autosynth [ERROR] > Synthesis failed
2021-08-28 08:52:20,445 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 6b21a68d7 docs: clarified some LRO types docs: fixed some wrong update mask descriptions
2021-08-28 08:52:20,517 autosynth [DEBUG] > Running: git checkout autosynth-Google.Cloud.Monitoring.V3
Switched to branch 'autosynth-Google.Cloud.Monitoring.V3'
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 359, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 191, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 339, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 68, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/f6da0a6c-6eec-48b3-8af3-7c3f6fb75b0a/targets/github%2Fsynthtool;config=default/tests;query=google-cloud-dotnet;failed=false).

Autosynth is still having trouble generating Google.Cloud.Monitoring.V3. :sob:

Please investigate and fix this issue within 5 business days.  While it remains broken,
this library cannot be updated with changes to the Google.Cloud.Monitoring.V3 API, and the library grows
stale.

See https://github.com/googleapis/synthtool/blob/master/autosynth/TroubleShooting.md
for trouble shooting tips.

Here's the output from running `synth.py`:

```
osynth [DEBUG] > Running: git log -1 --pretty=%at 061db160ed427e53884074d85061b59ce1702821
2021-08-28 22:50:20,668 autosynth [DEBUG] > Running: git log -1 --pretty=%at 7ae76714a266453c2e64a1faa75f156cbe7b6610
2021-08-28 22:50:20,670 autosynth [DEBUG] > Running: git log -1 --pretty=%at 9e617db5033d24bb8a47d6ac0d48de33b6f9ab16
2021-08-28 22:50:20,673 autosynth [DEBUG] > Running: git log -1 --pretty=%at ba30d8097582039ac4cc4e21b4e4baa426423075
2021-08-28 22:50:20,676 autosynth [DEBUG] > Running: git log -1 --pretty=%at 8c0e4a368056920eaa1039cd740a637c9d12cccb
2021-08-28 22:50:20,678 autosynth [DEBUG] > Running: git log -1 --pretty=%at 27600833c3d6e80cd676d5b54c54f89cb3f55665
2021-08-28 22:50:20,681 autosynth [DEBUG] > Running: git log -1 --pretty=%at cb637a6db2a0e406d656055a286ad921134c7d57
2021-08-28 22:50:20,684 autosynth [DEBUG] > Running: git log -1 --pretty=%at 8bf048db6962fffc98b3930de2b4044aa1d54957
2021-08-28 22:50:20,686 autosynth [DEBUG] > Running: git log -1 --pretty=%at 8a770c332426e30d7ce89de33d92eabd8835b8f0
2021-08-28 22:50:20,689 autosynth [DEBUG] > Running: git log -1 --pretty=%at 3cd708edbc97d8b0ceee6e667229e19f19249806
2021-08-28 22:50:20,692 autosynth [DEBUG] > Running: git log -1 --pretty=%at dd5050968965c901275e2d34d7e928c57028dfeb
2021-08-28 22:50:20,694 autosynth [DEBUG] > Running: git log -1 --pretty=%at cdc9747a73f6af2393ab51490fc3a3c11ff3620f
2021-08-28 22:50:20,697 autosynth [DEBUG] > Running: git log -1 --pretty=%at be31a88c169ec48f0862eb7632d0d514327d9948
2021-08-28 22:50:20,700 autosynth [DEBUG] > Running: git log -1 --pretty=%at 41e44ac10040bf2801c9a5c7e846e55029a02243
2021-08-28 22:50:20,703 autosynth [DEBUG] > Running: git log -1 --pretty=%at 5572865403528e706cd895b8817f0e54fae74c02
2021-08-28 22:50:20,705 autosynth [DEBUG] > Running: git log -1 --pretty=%at 410f323cdbbe047bbf4527ab453b89930917f4a8
2021-08-28 22:50:20,708 autosynth [DEBUG] > Running: git log -1 --pretty=%at 4006aa5cbd12a86550d45fb215511a98a697f48f
2021-08-28 22:50:20,710 autosynth [DEBUG] > Running: git log -1 --pretty=%at 4b6bb3dede843b93353dbebfee4d2758aeede96f
2021-08-28 22:50:20,713 autosynth [DEBUG] > Running: git log -1 --pretty=%at 9e36f4ec3accbe70c7d11e8242cf5f4e2a1e0957
2021-08-28 22:50:20,716 autosynth [DEBUG] > Running: git log -1 --pretty=%at 354d0763e7e4ce7ab370790ffba0c3c25b1420c3
2021-08-28 22:50:20,718 autosynth [DEBUG] > Running: git log -1 --pretty=%at ccb73479aebb76f13698907884269bb7a38b4207
2021-08-28 22:50:20,721 autosynth [DEBUG] > Running: git log -1 --pretty=%at 848854523c886fbfb421096a550533be96e0a49d
2021-08-28 22:50:20,724 autosynth [DEBUG] > Running: git log -1 --pretty=%at 43095748d60f2c54d908a27c420fef8ba3c9dbeb
2021-08-28 22:50:20,726 autosynth [DEBUG] > Running: git log -1 --pretty=%at 677caf0b524ea611211f39269bd1ee124438fba7
2021-08-28 22:50:20,729 autosynth [DEBUG] > Running: git log -1 --pretty=%at 41635ad2f614a85ac6ef92605b3e90d051b11ac0
2021-08-28 22:50:20,732 autosynth [DEBUG] > Running: git log -1 --pretty=%at e9da6f8b469c52b83f900e820be30762e9e05c57
2021-08-28 22:50:20,735 autosynth [DEBUG] > Running: git checkout 6b21a68d70a8bf416d2da830ca475a09d3e193b2
Note: switching to '6b21a68d70a8bf416d2da830ca475a09d3e193b2'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at 6b21a68d7 docs: clarified some LRO types docs: fixed some wrong update mask descriptions
2021-08-28 22:50:20,754 autosynth [DEBUG] > Running: git checkout e9da6f8b469c52b83f900e820be30762e9e05c57
Note: switching to 'e9da6f8b469c52b83f900e820be30762e9e05c57'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at e9da6f8b feat: support version reporting API
2021-08-28 22:50:20,767 autosynth [DEBUG] > Running: git branch -f autosynth-Google.Cloud.Monitoring.V3-47
2021-08-28 22:50:20,770 autosynth [DEBUG] > Running: git checkout autosynth-Google.Cloud.Monitoring.V3-47
Switched to branch 'autosynth-Google.Cloud.Monitoring.V3-47'
2021-08-28 22:50:20,789 autosynth [INFO] > Running synthtool
2021-08-28 22:50:20,789 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2021-08-28 22:50:20,789 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/Google.Cloud.Monitoring.V3/47/sponge_log.log
2021-08-28 22:50:20,791 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2021-08-28 22:50:20,991 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/google-cloud-dotnet/apis/Google.Cloud.Monitoring.V3/synth.py.
On branch autosynth-Google.Cloud.Monitoring.V3-47
nothing to commit, working tree clean
Skipping microgenerator fetch/build: already built, and running on Kokoro
Building existing version of Google.Cloud.Monitoring.V3 for compatibility checking
Generating Google.Cloud.Monitoring.V3
Building new version of Google.Cloud.Monitoring.V3 for compatibility checking
UptimeService.g.cs(93,245): error CS0234: The type or namespace name 'LongRunning' does not exist in the namespace 'Google' (are you missing an assembly reference?) [/home/kbuilder/.cache/synthtool/google-cloud-dotnet/apis/Google.Cloud.Monitoring.V3/Google.Cloud.Monitoring.V3/Google.Cloud.Monitoring.V3.csproj]
2021-08-28 22:50:37,694 synthtool [ERROR] > Failed executing /bin/bash generateapis.sh --check_compatibility Google.Cloud.Monitoring.V3:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1137, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1062, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 763, in invoke
    return __callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/google-cloud-dotnet/apis/Google.Cloud.Monitoring.V3/synth.py", line 26, in <module>
    hide_output = False)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '('/bin/bash', 'generateapis.sh', '--check_compatibility', 'Google.Cloud.Monitoring.V3')' returned non-zero exit status 1.
2021-08-28 22:50:37,719 autosynth [ERROR] > Synthesis failed
2021-08-28 22:50:37,719 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 6b21a68d7 docs: clarified some LRO types docs: fixed some wrong update mask descriptions
2021-08-28 22:50:37,774 autosynth [DEBUG] > Running: git checkout autosynth-Google.Cloud.Monitoring.V3
Switched to branch 'autosynth-Google.Cloud.Monitoring.V3'
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 359, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 191, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 339, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 68, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/3f8aa838-506c-4ba6-b25f-7791d6a029ab/targets/github%2Fsynthtool;config=default/tests;query=google-cloud-dotnet;failed=false).

Autosynth is still having trouble generating Google.Cloud.Monitoring.V3. :sob:

Please investigate and fix this issue within 5 business days.  While it remains broken,
this library cannot be updated with changes to the Google.Cloud.Monitoring.V3 API, and the library grows
stale.

See https://github.com/googleapis/synthtool/blob/master/autosynth/TroubleShooting.md
for trouble shooting tips.

Here's the output from running `synth.py`:

```
osynth [DEBUG] > Running: git log -1 --pretty=%at 061db160ed427e53884074d85061b59ce1702821
2021-08-29 08:53:01,071 autosynth [DEBUG] > Running: git log -1 --pretty=%at 7ae76714a266453c2e64a1faa75f156cbe7b6610
2021-08-29 08:53:01,074 autosynth [DEBUG] > Running: git log -1 --pretty=%at 9e617db5033d24bb8a47d6ac0d48de33b6f9ab16
2021-08-29 08:53:01,077 autosynth [DEBUG] > Running: git log -1 --pretty=%at ba30d8097582039ac4cc4e21b4e4baa426423075
2021-08-29 08:53:01,080 autosynth [DEBUG] > Running: git log -1 --pretty=%at 8c0e4a368056920eaa1039cd740a637c9d12cccb
2021-08-29 08:53:01,083 autosynth [DEBUG] > Running: git log -1 --pretty=%at 27600833c3d6e80cd676d5b54c54f89cb3f55665
2021-08-29 08:53:01,087 autosynth [DEBUG] > Running: git log -1 --pretty=%at cb637a6db2a0e406d656055a286ad921134c7d57
2021-08-29 08:53:01,090 autosynth [DEBUG] > Running: git log -1 --pretty=%at 8bf048db6962fffc98b3930de2b4044aa1d54957
2021-08-29 08:53:01,093 autosynth [DEBUG] > Running: git log -1 --pretty=%at 8a770c332426e30d7ce89de33d92eabd8835b8f0
2021-08-29 08:53:01,097 autosynth [DEBUG] > Running: git log -1 --pretty=%at 3cd708edbc97d8b0ceee6e667229e19f19249806
2021-08-29 08:53:01,100 autosynth [DEBUG] > Running: git log -1 --pretty=%at dd5050968965c901275e2d34d7e928c57028dfeb
2021-08-29 08:53:01,103 autosynth [DEBUG] > Running: git log -1 --pretty=%at cdc9747a73f6af2393ab51490fc3a3c11ff3620f
2021-08-29 08:53:01,106 autosynth [DEBUG] > Running: git log -1 --pretty=%at be31a88c169ec48f0862eb7632d0d514327d9948
2021-08-29 08:53:01,110 autosynth [DEBUG] > Running: git log -1 --pretty=%at 41e44ac10040bf2801c9a5c7e846e55029a02243
2021-08-29 08:53:01,113 autosynth [DEBUG] > Running: git log -1 --pretty=%at 5572865403528e706cd895b8817f0e54fae74c02
2021-08-29 08:53:01,116 autosynth [DEBUG] > Running: git log -1 --pretty=%at 410f323cdbbe047bbf4527ab453b89930917f4a8
2021-08-29 08:53:01,120 autosynth [DEBUG] > Running: git log -1 --pretty=%at 4006aa5cbd12a86550d45fb215511a98a697f48f
2021-08-29 08:53:01,123 autosynth [DEBUG] > Running: git log -1 --pretty=%at 4b6bb3dede843b93353dbebfee4d2758aeede96f
2021-08-29 08:53:01,126 autosynth [DEBUG] > Running: git log -1 --pretty=%at 9e36f4ec3accbe70c7d11e8242cf5f4e2a1e0957
2021-08-29 08:53:01,130 autosynth [DEBUG] > Running: git log -1 --pretty=%at 354d0763e7e4ce7ab370790ffba0c3c25b1420c3
2021-08-29 08:53:01,133 autosynth [DEBUG] > Running: git log -1 --pretty=%at ccb73479aebb76f13698907884269bb7a38b4207
2021-08-29 08:53:01,136 autosynth [DEBUG] > Running: git log -1 --pretty=%at 848854523c886fbfb421096a550533be96e0a49d
2021-08-29 08:53:01,139 autosynth [DEBUG] > Running: git log -1 --pretty=%at 43095748d60f2c54d908a27c420fef8ba3c9dbeb
2021-08-29 08:53:01,143 autosynth [DEBUG] > Running: git log -1 --pretty=%at 677caf0b524ea611211f39269bd1ee124438fba7
2021-08-29 08:53:01,146 autosynth [DEBUG] > Running: git log -1 --pretty=%at 41635ad2f614a85ac6ef92605b3e90d051b11ac0
2021-08-29 08:53:01,149 autosynth [DEBUG] > Running: git log -1 --pretty=%at e9da6f8b469c52b83f900e820be30762e9e05c57
2021-08-29 08:53:01,153 autosynth [DEBUG] > Running: git checkout 6b21a68d70a8bf416d2da830ca475a09d3e193b2
Note: switching to '6b21a68d70a8bf416d2da830ca475a09d3e193b2'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at 6b21a68d7 docs: clarified some LRO types docs: fixed some wrong update mask descriptions
2021-08-29 08:53:01,176 autosynth [DEBUG] > Running: git checkout e9da6f8b469c52b83f900e820be30762e9e05c57
Note: switching to 'e9da6f8b469c52b83f900e820be30762e9e05c57'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at e9da6f8b feat: support version reporting API
2021-08-29 08:53:01,192 autosynth [DEBUG] > Running: git branch -f autosynth-Google.Cloud.Monitoring.V3-47
2021-08-29 08:53:01,196 autosynth [DEBUG] > Running: git checkout autosynth-Google.Cloud.Monitoring.V3-47
Switched to branch 'autosynth-Google.Cloud.Monitoring.V3-47'
2021-08-29 08:53:01,220 autosynth [INFO] > Running synthtool
2021-08-29 08:53:01,220 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2021-08-29 08:53:01,221 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/Google.Cloud.Monitoring.V3/47/sponge_log.log
2021-08-29 08:53:01,223 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2021-08-29 08:53:01,464 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/google-cloud-dotnet/apis/Google.Cloud.Monitoring.V3/synth.py.
On branch autosynth-Google.Cloud.Monitoring.V3-47
nothing to commit, working tree clean
Skipping microgenerator fetch/build: already built, and running on Kokoro
Building existing version of Google.Cloud.Monitoring.V3 for compatibility checking
Generating Google.Cloud.Monitoring.V3
Building new version of Google.Cloud.Monitoring.V3 for compatibility checking
UptimeService.g.cs(93,245): error CS0234: The type or namespace name 'LongRunning' does not exist in the namespace 'Google' (are you missing an assembly reference?) [/home/kbuilder/.cache/synthtool/google-cloud-dotnet/apis/Google.Cloud.Monitoring.V3/Google.Cloud.Monitoring.V3/Google.Cloud.Monitoring.V3.csproj]
2021-08-29 08:53:20,501 synthtool [ERROR] > Failed executing /bin/bash generateapis.sh --check_compatibility Google.Cloud.Monitoring.V3:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1137, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1062, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 763, in invoke
    return __callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/google-cloud-dotnet/apis/Google.Cloud.Monitoring.V3/synth.py", line 26, in <module>
    hide_output = False)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '('/bin/bash', 'generateapis.sh', '--check_compatibility', 'Google.Cloud.Monitoring.V3')' returned non-zero exit status 1.
2021-08-29 08:53:20,533 autosynth [ERROR] > Synthesis failed
2021-08-29 08:53:20,534 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 6b21a68d7 docs: clarified some LRO types docs: fixed some wrong update mask descriptions
2021-08-29 08:53:20,612 autosynth [DEBUG] > Running: git checkout autosynth-Google.Cloud.Monitoring.V3
Switched to branch 'autosynth-Google.Cloud.Monitoring.V3'
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 359, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 191, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 339, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 68, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/5a61baa3-e07e-485f-ab50-0b90dba71cdd/targets/github%2Fsynthtool;config=default/tests;query=google-cloud-dotnet;failed=false).

Autosynth is still having trouble generating Google.Cloud.Monitoring.V3. :sob:

Please investigate and fix this issue within 5 business days.  While it remains broken,
this library cannot be updated with changes to the Google.Cloud.Monitoring.V3 API, and the library grows
stale.

See https://github.com/googleapis/synthtool/blob/master/autosynth/TroubleShooting.md
for trouble shooting tips.

Here's the output from running `synth.py`:

```
,579 autosynth [DEBUG] > Running: git log -1 --pretty=%at 9e617db5033d24bb8a47d6ac0d48de33b6f9ab16
2021-08-29 22:50:46,583 autosynth [DEBUG] > Running: git log -1 --pretty=%at ba30d8097582039ac4cc4e21b4e4baa426423075
2021-08-29 22:50:46,585 autosynth [DEBUG] > Running: git log -1 --pretty=%at 8c0e4a368056920eaa1039cd740a637c9d12cccb
2021-08-29 22:50:46,588 autosynth [DEBUG] > Running: git log -1 --pretty=%at 27600833c3d6e80cd676d5b54c54f89cb3f55665
2021-08-29 22:50:46,591 autosynth [DEBUG] > Running: git log -1 --pretty=%at cb637a6db2a0e406d656055a286ad921134c7d57
2021-08-29 22:50:46,594 autosynth [DEBUG] > Running: git log -1 --pretty=%at 8bf048db6962fffc98b3930de2b4044aa1d54957
2021-08-29 22:50:46,597 autosynth [DEBUG] > Running: git log -1 --pretty=%at 8a770c332426e30d7ce89de33d92eabd8835b8f0
2021-08-29 22:50:46,600 autosynth [DEBUG] > Running: git log -1 --pretty=%at 3cd708edbc97d8b0ceee6e667229e19f19249806
2021-08-29 22:50:46,602 autosynth [DEBUG] > Running: git log -1 --pretty=%at dd5050968965c901275e2d34d7e928c57028dfeb
2021-08-29 22:50:46,606 autosynth [DEBUG] > Running: git log -1 --pretty=%at cdc9747a73f6af2393ab51490fc3a3c11ff3620f
2021-08-29 22:50:46,608 autosynth [DEBUG] > Running: git log -1 --pretty=%at be31a88c169ec48f0862eb7632d0d514327d9948
2021-08-29 22:50:46,611 autosynth [DEBUG] > Running: git log -1 --pretty=%at 41e44ac10040bf2801c9a5c7e846e55029a02243
2021-08-29 22:50:46,614 autosynth [DEBUG] > Running: git log -1 --pretty=%at 5572865403528e706cd895b8817f0e54fae74c02
2021-08-29 22:50:46,617 autosynth [DEBUG] > Running: git log -1 --pretty=%at 410f323cdbbe047bbf4527ab453b89930917f4a8
2021-08-29 22:50:46,620 autosynth [DEBUG] > Running: git log -1 --pretty=%at 4006aa5cbd12a86550d45fb215511a98a697f48f
2021-08-29 22:50:46,622 autosynth [DEBUG] > Running: git log -1 --pretty=%at 4b6bb3dede843b93353dbebfee4d2758aeede96f
2021-08-29 22:50:46,625 autosynth [DEBUG] > Running: git log -1 --pretty=%at 9e36f4ec3accbe70c7d11e8242cf5f4e2a1e0957
2021-08-29 22:50:46,628 autosynth [DEBUG] > Running: git log -1 --pretty=%at 354d0763e7e4ce7ab370790ffba0c3c25b1420c3
2021-08-29 22:50:46,631 autosynth [DEBUG] > Running: git log -1 --pretty=%at ccb73479aebb76f13698907884269bb7a38b4207
2021-08-29 22:50:46,633 autosynth [DEBUG] > Running: git log -1 --pretty=%at 848854523c886fbfb421096a550533be96e0a49d
2021-08-29 22:50:46,636 autosynth [DEBUG] > Running: git log -1 --pretty=%at 43095748d60f2c54d908a27c420fef8ba3c9dbeb
2021-08-29 22:50:46,639 autosynth [DEBUG] > Running: git log -1 --pretty=%at 677caf0b524ea611211f39269bd1ee124438fba7
2021-08-29 22:50:46,641 autosynth [DEBUG] > Running: git log -1 --pretty=%at 41635ad2f614a85ac6ef92605b3e90d051b11ac0
2021-08-29 22:50:46,644 autosynth [DEBUG] > Running: git log -1 --pretty=%at e9da6f8b469c52b83f900e820be30762e9e05c57
2021-08-29 22:50:46,647 autosynth [DEBUG] > Running: git log -1 --pretty=%at 17d3fb258cc9527ddcf910a72126de3a43fac440
2021-08-29 22:50:46,650 autosynth [DEBUG] > Running: git log -1 --pretty=%at ce550012218df52b529a942d599f7f7e68af0042
2021-08-29 22:50:46,653 autosynth [DEBUG] > Running: git checkout 6b21a68d70a8bf416d2da830ca475a09d3e193b2
Note: switching to '6b21a68d70a8bf416d2da830ca475a09d3e193b2'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at 6b21a68d7 docs: clarified some LRO types docs: fixed some wrong update mask descriptions
2021-08-29 22:50:46,673 autosynth [DEBUG] > Running: git checkout ce550012218df52b529a942d599f7f7e68af0042
Note: switching to 'ce550012218df52b529a942d599f7f7e68af0042'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at ce550012 chore: regenerate API index
2021-08-29 22:50:46,687 autosynth [DEBUG] > Running: git branch -f autosynth-Google.Cloud.Monitoring.V3-49
2021-08-29 22:50:46,689 autosynth [DEBUG] > Running: git checkout autosynth-Google.Cloud.Monitoring.V3-49
Switched to branch 'autosynth-Google.Cloud.Monitoring.V3-49'
2021-08-29 22:50:46,709 autosynth [INFO] > Running synthtool
2021-08-29 22:50:46,709 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2021-08-29 22:50:46,709 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/Google.Cloud.Monitoring.V3/49/sponge_log.log
2021-08-29 22:50:46,711 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2021-08-29 22:50:46,913 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/google-cloud-dotnet/apis/Google.Cloud.Monitoring.V3/synth.py.
On branch autosynth-Google.Cloud.Monitoring.V3-49
nothing to commit, working tree clean
Skipping microgenerator fetch/build: already built, and running on Kokoro
Building existing version of Google.Cloud.Monitoring.V3 for compatibility checking
Generating Google.Cloud.Monitoring.V3
Building new version of Google.Cloud.Monitoring.V3 for compatibility checking
UptimeService.g.cs(93,245): error CS0234: The type or namespace name 'LongRunning' does not exist in the namespace 'Google' (are you missing an assembly reference?) [/home/kbuilder/.cache/synthtool/google-cloud-dotnet/apis/Google.Cloud.Monitoring.V3/Google.Cloud.Monitoring.V3/Google.Cloud.Monitoring.V3.csproj]
2021-08-29 22:51:02,917 synthtool [ERROR] > Failed executing /bin/bash generateapis.sh --check_compatibility Google.Cloud.Monitoring.V3:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1137, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1062, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 763, in invoke
    return __callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/google-cloud-dotnet/apis/Google.Cloud.Monitoring.V3/synth.py", line 26, in <module>
    hide_output = False)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '('/bin/bash', 'generateapis.sh', '--check_compatibility', 'Google.Cloud.Monitoring.V3')' returned non-zero exit status 1.
2021-08-29 22:51:02,943 autosynth [ERROR] > Synthesis failed
2021-08-29 22:51:02,943 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 6b21a68d7 docs: clarified some LRO types docs: fixed some wrong update mask descriptions
2021-08-29 22:51:03,000 autosynth [DEBUG] > Running: git checkout autosynth-Google.Cloud.Monitoring.V3
Switched to branch 'autosynth-Google.Cloud.Monitoring.V3'
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 359, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 191, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 339, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 68, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/0bb4a792-bc6f-4dbc-b2fa-530f29bb0f3c/targets/github%2Fsynthtool;config=default/tests;query=google-cloud-dotnet;failed=false).

Autosynth is still having trouble generating Google.Cloud.Monitoring.V3. :sob:

Please investigate and fix this issue within 5 business days.  While it remains broken,
this library cannot be updated with changes to the Google.Cloud.Monitoring.V3 API, and the library grows
stale.

See https://github.com/googleapis/synthtool/blob/master/autosynth/TroubleShooting.md
for trouble shooting tips.

Here's the output from running `synth.py`:

```
a6db2a0e406d656055a286ad921134c7d57
2021-08-30 08:53:45,160 autosynth [DEBUG] > Running: git log -1 --pretty=%at 8bf048db6962fffc98b3930de2b4044aa1d54957
2021-08-30 08:53:45,163 autosynth [DEBUG] > Running: git log -1 --pretty=%at 8a770c332426e30d7ce89de33d92eabd8835b8f0
2021-08-30 08:53:45,166 autosynth [DEBUG] > Running: git log -1 --pretty=%at 3cd708edbc97d8b0ceee6e667229e19f19249806
2021-08-30 08:53:45,169 autosynth [DEBUG] > Running: git log -1 --pretty=%at dd5050968965c901275e2d34d7e928c57028dfeb
2021-08-30 08:53:45,172 autosynth [DEBUG] > Running: git log -1 --pretty=%at cdc9747a73f6af2393ab51490fc3a3c11ff3620f
2021-08-30 08:53:45,175 autosynth [DEBUG] > Running: git log -1 --pretty=%at be31a88c169ec48f0862eb7632d0d514327d9948
2021-08-30 08:53:45,178 autosynth [DEBUG] > Running: git log -1 --pretty=%at 41e44ac10040bf2801c9a5c7e846e55029a02243
2021-08-30 08:53:45,182 autosynth [DEBUG] > Running: git log -1 --pretty=%at 5572865403528e706cd895b8817f0e54fae74c02
2021-08-30 08:53:45,185 autosynth [DEBUG] > Running: git log -1 --pretty=%at 410f323cdbbe047bbf4527ab453b89930917f4a8
2021-08-30 08:53:45,188 autosynth [DEBUG] > Running: git log -1 --pretty=%at 4006aa5cbd12a86550d45fb215511a98a697f48f
2021-08-30 08:53:45,191 autosynth [DEBUG] > Running: git log -1 --pretty=%at 4b6bb3dede843b93353dbebfee4d2758aeede96f
2021-08-30 08:53:45,194 autosynth [DEBUG] > Running: git log -1 --pretty=%at 9e36f4ec3accbe70c7d11e8242cf5f4e2a1e0957
2021-08-30 08:53:45,196 autosynth [DEBUG] > Running: git log -1 --pretty=%at 354d0763e7e4ce7ab370790ffba0c3c25b1420c3
2021-08-30 08:53:45,199 autosynth [DEBUG] > Running: git log -1 --pretty=%at ccb73479aebb76f13698907884269bb7a38b4207
2021-08-30 08:53:45,202 autosynth [DEBUG] > Running: git log -1 --pretty=%at 848854523c886fbfb421096a550533be96e0a49d
2021-08-30 08:53:45,205 autosynth [DEBUG] > Running: git log -1 --pretty=%at 43095748d60f2c54d908a27c420fef8ba3c9dbeb
2021-08-30 08:53:45,208 autosynth [DEBUG] > Running: git log -1 --pretty=%at 677caf0b524ea611211f39269bd1ee124438fba7
2021-08-30 08:53:45,211 autosynth [DEBUG] > Running: git log -1 --pretty=%at 41635ad2f614a85ac6ef92605b3e90d051b11ac0
2021-08-30 08:53:45,214 autosynth [DEBUG] > Running: git log -1 --pretty=%at e9da6f8b469c52b83f900e820be30762e9e05c57
2021-08-30 08:53:45,216 autosynth [DEBUG] > Running: git log -1 --pretty=%at 17d3fb258cc9527ddcf910a72126de3a43fac440
2021-08-30 08:53:45,219 autosynth [DEBUG] > Running: git log -1 --pretty=%at ce550012218df52b529a942d599f7f7e68af0042
2021-08-30 08:53:45,222 autosynth [DEBUG] > Running: git log -1 --pretty=%at 9e99074fc73529d946fca7914deb0d21968f03d8
2021-08-30 08:53:45,225 autosynth [DEBUG] > Running: git log -1 --pretty=%at 69fe13f7ca3ca96f34e042596c11e40bb3277924
2021-08-30 08:53:45,228 autosynth [DEBUG] > Running: git log -1 --pretty=%at 577814ea106e5f5514e8779f72749b9cfefa54e4
2021-08-30 08:53:45,231 autosynth [DEBUG] > Running: git log -1 --pretty=%at e3d9b11abaec81e1c3c9c7cab1f3bb6fa3990651
2021-08-30 08:53:45,234 autosynth [DEBUG] > Running: git checkout 1440a7c024e9b5e9e511513aa17a3737aca99fa7
Note: switching to '1440a7c024e9b5e9e511513aa17a3737aca99fa7'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at 1440a7c02 feat: added v3 proto for online/batch document translation and updated v3beta1 proto for format conversion
2021-08-30 08:53:45,255 autosynth [DEBUG] > Running: git checkout e3d9b11abaec81e1c3c9c7cab1f3bb6fa3990651
Note: switching to 'e3d9b11abaec81e1c3c9c7cab1f3bb6fa3990651'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at e3d9b11a feat: add support for Key Reimport
2021-08-30 08:53:45,268 autosynth [DEBUG] > Running: git branch -f autosynth-Google.Cloud.Monitoring.V3-53
2021-08-30 08:53:45,271 autosynth [DEBUG] > Running: git checkout autosynth-Google.Cloud.Monitoring.V3-53
Switched to branch 'autosynth-Google.Cloud.Monitoring.V3-53'
2021-08-30 08:53:45,292 autosynth [INFO] > Running synthtool
2021-08-30 08:53:45,292 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2021-08-30 08:53:45,292 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/Google.Cloud.Monitoring.V3/53/sponge_log.log
2021-08-30 08:53:45,294 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2021-08-30 08:53:45,501 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/google-cloud-dotnet/apis/Google.Cloud.Monitoring.V3/synth.py.
On branch autosynth-Google.Cloud.Monitoring.V3-53
nothing to commit, working tree clean
Skipping microgenerator fetch/build: already built, and running on Kokoro
Building existing version of Google.Cloud.Monitoring.V3 for compatibility checking
Generating Google.Cloud.Monitoring.V3
Building new version of Google.Cloud.Monitoring.V3 for compatibility checking
UptimeService.g.cs(93,245): error CS0234: The type or namespace name 'LongRunning' does not exist in the namespace 'Google' (are you missing an assembly reference?) [/home/kbuilder/.cache/synthtool/google-cloud-dotnet/apis/Google.Cloud.Monitoring.V3/Google.Cloud.Monitoring.V3/Google.Cloud.Monitoring.V3.csproj]
2021-08-30 08:54:02,121 synthtool [ERROR] > Failed executing /bin/bash generateapis.sh --check_compatibility Google.Cloud.Monitoring.V3:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1137, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1062, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 763, in invoke
    return __callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/google-cloud-dotnet/apis/Google.Cloud.Monitoring.V3/synth.py", line 26, in <module>
    hide_output = False)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '('/bin/bash', 'generateapis.sh', '--check_compatibility', 'Google.Cloud.Monitoring.V3')' returned non-zero exit status 1.
2021-08-30 08:54:02,148 autosynth [ERROR] > Synthesis failed
2021-08-30 08:54:02,148 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 1440a7c02 feat: added v3 proto for online/batch document translation and updated v3beta1 proto for format conversion
2021-08-30 08:54:02,209 autosynth [DEBUG] > Running: git checkout autosynth-Google.Cloud.Monitoring.V3
Switched to branch 'autosynth-Google.Cloud.Monitoring.V3'
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 359, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 191, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 339, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 68, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/ae27bed1-eda9-4f61-ad13-ee30fede172b/targets/github%2Fsynthtool;config=default/tests;query=google-cloud-dotnet;failed=false).

Autosynth is still having trouble generating Google.Cloud.Monitoring.V3. :sob:

Please investigate and fix this issue within 5 business days.  While it remains broken,
this library cannot be updated with changes to the Google.Cloud.Monitoring.V3 API, and the library grows
stale.

See https://github.com/googleapis/synthtool/blob/master/autosynth/TroubleShooting.md
for trouble shooting tips.

Here's the output from running `synth.py`:

```
817f0e54fae74c02
2021-08-30 22:57:43,175 autosynth [DEBUG] > Running: git log -1 --pretty=%at 410f323cdbbe047bbf4527ab453b89930917f4a8
2021-08-30 22:57:43,180 autosynth [DEBUG] > Running: git log -1 --pretty=%at 4006aa5cbd12a86550d45fb215511a98a697f48f
2021-08-30 22:57:43,184 autosynth [DEBUG] > Running: git log -1 --pretty=%at 4b6bb3dede843b93353dbebfee4d2758aeede96f
2021-08-30 22:57:43,189 autosynth [DEBUG] > Running: git log -1 --pretty=%at 9e36f4ec3accbe70c7d11e8242cf5f4e2a1e0957
2021-08-30 22:57:43,193 autosynth [DEBUG] > Running: git log -1 --pretty=%at 354d0763e7e4ce7ab370790ffba0c3c25b1420c3
2021-08-30 22:57:43,198 autosynth [DEBUG] > Running: git log -1 --pretty=%at ccb73479aebb76f13698907884269bb7a38b4207
2021-08-30 22:57:43,202 autosynth [DEBUG] > Running: git log -1 --pretty=%at 848854523c886fbfb421096a550533be96e0a49d
2021-08-30 22:57:43,207 autosynth [DEBUG] > Running: git log -1 --pretty=%at 43095748d60f2c54d908a27c420fef8ba3c9dbeb
2021-08-30 22:57:43,211 autosynth [DEBUG] > Running: git log -1 --pretty=%at 677caf0b524ea611211f39269bd1ee124438fba7
2021-08-30 22:57:43,216 autosynth [DEBUG] > Running: git log -1 --pretty=%at 41635ad2f614a85ac6ef92605b3e90d051b11ac0
2021-08-30 22:57:43,221 autosynth [DEBUG] > Running: git log -1 --pretty=%at e9da6f8b469c52b83f900e820be30762e9e05c57
2021-08-30 22:57:43,225 autosynth [DEBUG] > Running: git log -1 --pretty=%at 17d3fb258cc9527ddcf910a72126de3a43fac440
2021-08-30 22:57:43,229 autosynth [DEBUG] > Running: git log -1 --pretty=%at ce550012218df52b529a942d599f7f7e68af0042
2021-08-30 22:57:43,234 autosynth [DEBUG] > Running: git log -1 --pretty=%at 9e99074fc73529d946fca7914deb0d21968f03d8
2021-08-30 22:57:43,239 autosynth [DEBUG] > Running: git log -1 --pretty=%at 69fe13f7ca3ca96f34e042596c11e40bb3277924
2021-08-30 22:57:43,243 autosynth [DEBUG] > Running: git log -1 --pretty=%at 577814ea106e5f5514e8779f72749b9cfefa54e4
2021-08-30 22:57:43,247 autosynth [DEBUG] > Running: git log -1 --pretty=%at e3d9b11abaec81e1c3c9c7cab1f3bb6fa3990651
2021-08-30 22:57:43,252 autosynth [DEBUG] > Running: git log -1 --pretty=%at c1296cd1db77b40f87fa95f47c58cd5ea13e66db
2021-08-30 22:57:43,257 autosynth [DEBUG] > Running: git log -1 --pretty=%at 03ce2ebf5a3d7753ad074bfe53acea5c136748ce
2021-08-30 22:57:43,262 autosynth [DEBUG] > Running: git log -1 --pretty=%at 583dd6a542a82295e593bfd908d2c56abf8dad46
2021-08-30 22:57:43,267 autosynth [DEBUG] > Running: git log -1 --pretty=%at 8031cc26fe7f1163faf8778323c8762541c0aa6b
2021-08-30 22:57:43,272 autosynth [DEBUG] > Running: git log -1 --pretty=%at 2e9e9ef336d59d882c0e770402c405d62ac83028
2021-08-30 22:57:43,277 autosynth [DEBUG] > Running: git log -1 --pretty=%at fbae10c8f7c9c6414af52f6499ffcc94e1170678
2021-08-30 22:57:43,282 autosynth [DEBUG] > Running: git log -1 --pretty=%at 298a980ac0d20a806cda281912db05c1d9a1e731
2021-08-30 22:57:43,287 autosynth [DEBUG] > Running: git log -1 --pretty=%at 321abab2143fed12030b32b2b20aecdc3a5070ed
2021-08-30 22:57:43,292 autosynth [DEBUG] > Running: git log -1 --pretty=%at 2127126597ea49fb0f5a5c23a2223312f3c32835
2021-08-30 22:57:43,298 autosynth [DEBUG] > Running: git checkout c38ba1fdc083925178141b880a28a5c15b2181b8
Note: switching to 'c38ba1fdc083925178141b880a28a5c15b2181b8'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at c38ba1fdc feat: Update osconfig v1 and v1alpha with WindowsApplication
2021-08-30 22:57:43,325 autosynth [DEBUG] > Running: git checkout 2127126597ea49fb0f5a5c23a2223312f3c32835
Note: switching to '2127126597ea49fb0f5a5c23a2223312f3c32835'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at 21271265 chore: regenerate API index
2021-08-30 22:57:43,343 autosynth [DEBUG] > Running: git branch -f autosynth-Google.Cloud.Monitoring.V3-62
2021-08-30 22:57:43,348 autosynth [DEBUG] > Running: git checkout autosynth-Google.Cloud.Monitoring.V3-62
Switched to branch 'autosynth-Google.Cloud.Monitoring.V3-62'
2021-08-30 22:57:43,376 autosynth [INFO] > Running synthtool
2021-08-30 22:57:43,376 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2021-08-30 22:57:43,376 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/Google.Cloud.Monitoring.V3/62/sponge_log.log
2021-08-30 22:57:43,379 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2021-08-30 22:57:43,627 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/google-cloud-dotnet/apis/Google.Cloud.Monitoring.V3/synth.py.
On branch autosynth-Google.Cloud.Monitoring.V3-62
nothing to commit, working tree clean
Skipping microgenerator fetch/build: already built, and running on Kokoro
Building existing version of Google.Cloud.Monitoring.V3 for compatibility checking
Generating Google.Cloud.Monitoring.V3
Building new version of Google.Cloud.Monitoring.V3 for compatibility checking
UptimeService.g.cs(93,245): error CS0234: The type or namespace name 'LongRunning' does not exist in the namespace 'Google' (are you missing an assembly reference?) [/home/kbuilder/.cache/synthtool/google-cloud-dotnet/apis/Google.Cloud.Monitoring.V3/Google.Cloud.Monitoring.V3/Google.Cloud.Monitoring.V3.csproj]
2021-08-30 22:58:03,614 synthtool [ERROR] > Failed executing /bin/bash generateapis.sh --check_compatibility Google.Cloud.Monitoring.V3:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1137, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1062, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 763, in invoke
    return __callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/google-cloud-dotnet/apis/Google.Cloud.Monitoring.V3/synth.py", line 26, in <module>
    hide_output = False)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '('/bin/bash', 'generateapis.sh', '--check_compatibility', 'Google.Cloud.Monitoring.V3')' returned non-zero exit status 1.
2021-08-30 22:58:03,646 autosynth [ERROR] > Synthesis failed
2021-08-30 22:58:03,646 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at c38ba1fdc feat: Update osconfig v1 and v1alpha with WindowsApplication
2021-08-30 22:58:03,726 autosynth [DEBUG] > Running: git checkout autosynth-Google.Cloud.Monitoring.V3
Switched to branch 'autosynth-Google.Cloud.Monitoring.V3'
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 359, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 191, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 339, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 68, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/9971e792-d831-40cb-bf39-be9486fcd99e/targets/github%2Fsynthtool;config=default/tests;query=google-cloud-dotnet;failed=false).

--- a/apis/Google.Cloud.Monitoring.V3/Google.Cloud.Monitoring.V3/Google.Cloud.Monitoring.V3.csproj
+++ b/apis/Google.Cloud.Monitoring.V3/Google.Cloud.Monitoring.V3/Google.Cloud.Monitoring.V3.csproj
@@ -12,6 +12,7 @@
     <PackageReference Include="ConfigureAwaitChecker.Analyzer" Version="5.0.0" PrivateAssets="All" />
     <PackageReference Include="Google.Api.CommonProtos" Version="[2.3.0, 3.0.0)" />
     <PackageReference Include="Google.Api.Gax.Grpc.GrpcCore" Version="[3.5.0, 4.0.0)" />
+    <PackageReference Include="Google.LongRunning" Version="[2.2.0, 3.0.0)" />
     <PackageReference Include="Grpc.Core" Version="[2.38.1, 3.0.0)" PrivateAssets="None" />
     <PackageReference Include="Microsoft.DotNet.Analyzers.Compatibility" Version="0.2.12-alpha" PrivateAssets="All" />
     <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.2" PrivateAssets="All" />

--- a/apis/apis.json
+++ b/apis/apis.json
@@ -1702,6 +1702,7 @@
       "dependencies": {
         "Google.Api.CommonProtos": "2.3.0",
         "Google.Api.Gax.Grpc.GrpcCore": "3.5.0",
+        "Google.LongRunning": "2.2.0",
         "Grpc.Core": "2.38.1"
       }
     },

